fcrepo-configs/src/main/java/org/fcrepo/config/FedoraPropsConfig.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.fcrepo.config;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+import javax.annotation.PostConstruct;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+/**
+ * General Fedora properties
+ */
+@Configuration
+public class FedoraPropsConfig {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(FedoraPropsConfig.class);
+
+    private static final String DATA_DIR = "data";
+
+    @Value("${fcrepo.home:fcrepo}")
+    private Path fedoraHome;
+
+    @Value("#{fedoraPropsConfig.fedoraHome.resolve('" + DATA_DIR + "')}")
```

Comment:
Have you tested these with IntelliJ and Eclipse? It would be good to get @bbpennel 's sign-off.

Reply:
Have not tested in Eclipse